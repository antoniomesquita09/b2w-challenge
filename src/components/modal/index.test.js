import React from 'react'
import { render } from '@testing-library/react'

import Modal from './index.js'

describe('<Modal />', () => {
  it('Snapshot testing Modal', () => {
    const { asFragment } = render(<Modal />)

    expect(asFragment()).toMatchSnapshot()
  })
})
