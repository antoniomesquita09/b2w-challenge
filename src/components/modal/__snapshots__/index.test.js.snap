// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal /> Snapshot testing Modal 1`] = `
<DocumentFragment>
  <div
    class="root"
  />
</DocumentFragment>
`;
